haiku-apps/beae/beae-1.2.recipe

```diff
@@ -4,12 +4,19 @@ in/out, analyzing, etc."
 HOMEPAGE="https://github.com/HaikuArchives/BeAE"
 COPYRIGHT="2003 Xentronix"
 LICENSE="BSD (3-clause)"
-REVISION="3"
-SOURCE_URI="$(HOMEPAGE)/archive/a90a9872420afe1a06537c798bfdfdc36d631323.tar.gz"
-SOURCE_DIR="beae-1.2"
-ARCHITECTURES="x86_gcc2 !x86 ?x86_64"
-SECONDARY_ARCHITECTURES="!x86"
+REVISION="4"
+SOURCE_URI="https://github.com/HaikuArchives/BeAE/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="d69a16821f86eda6e94b8297928f9def75f5c750e1e7d6b946d0b195fdc88943"
+SOURCE_FILENAME="BeAE-$portVersion.tar.gz"
+SOURCE_DIR="BeAE-$portVersion"
 
+ARCHITECTURES="x86_gcc2 x86 ?x86_64"
```

Comment:
Should be changed to:
```
ARCHITECTURES="x86_gcc2 x86 !x86_64" 
```
(since the build still fails on x86_64)

> Ok, I just made the suggested changes. But it needs to be rechecked to make sure it still builds on x86_gcc2 and x86 and can build secondary on each.

You are right! BeAE now also builds fine on x86 ;-)
And on x86_gcc2 I can build beae and beae_x86.

So this means we can probably drop support for secondary architectures.

BTW, we only need one PROVIDES ;-)